crates/solidity/testing/snapshots/cst_output/Expression/function_call_options_split/generated/0.8.0.yml

```diff
@@ -5,12 +5,20 @@ Source: >
 
 Errors: # 1 total
   - >
-    Error: Expected end of input.
+    Error: Expected .
        ╭─[crates/solidity/testing/snapshots/cst_output/Expression/function_call_options_split/input.sol:1:4]
        │
      1 │ a.b{value: 0}{gas: 1}("")
-       │    │ 
-       │    ╰─ Error occurred here.
+       │    ───────────┬──────────  
+       │               ╰──────────── Error occurred here.
     ───╯
 
-Tree: null
+Tree:
+  - Expression (Rule): # 0..25 'a.b{value: 0}{gas: 1}("")'
+      - MemberAccessExpression (Rule): # 0..3 "a.b"
```

Comment:
Earlier, we discovered a bug where operators don't wrap their operands with an additional base `Expression` node. This is correct if `Expression` is inlined, but in this case here, it is not, and therefore, they should be wrapped.
Otherwise, users won't be able to visit all expressions recusively.

IIRC, it was getting fixed in this PR, is that still the plan?
Definetely not blocking if not.